benchopt/utils/conda_env_cmd.py

```diff
@@ -155,11 +155,23 @@ def get_benchopt_version_in_env(env_name):
         "benchopt --version --check-editable",
         env_name=env_name, capture_stdout=True, return_output=True
     )
+
     if check_benchopt != 0:
         if DEBUG:
             print(output)
         return None, None
     benchopt_version, is_editable = output.split()
+
+    # check against running version
+    _, output = _run_shell_in_conda_env(
+        "benchopt --version --check-editable",
+        env_name=None, capture_stdout=True, return_output=True
+    )
+    version_running = output.split()[0]
```

Comment:
You can access this without using a subcommand that is a bit slow here:
```suggestion
    from benchopt import __version__ as version_running
```

Reply:
Is there a way to check that it's editable in both envs, without the call to `benchopt --version` ?